### DISCUSSION TITLE
Formatting for table columns

### DISCUSSION THREAD

**Discussion by phangs** [Budibase/budibase#1597] [Ideas]
Using tables is one very fast way to show information to the users using data provider.  Currently, the table component does not allow formatting of the columns, it would be great if we can add formatting to the column data. For example, the data shows a number/currency value, the formatting will allow us to right align the value as well as add decimals to it, great if the number of decimals and comma can be specified.  Same with datetime values, we can change the data format to display only the date and not include the time, would be better if we have several options how we can display the date value.

So basically, number and date formatting and alignment for specific columns would be a great for displaying data in a table.

**Comment by davidmabelle** [comment]
Also need to be able to set font size and font family.

**Reply by mjashanks** [reply]
Yes, we get asked for updates to the table component a lot and hope to tackle it soon: https://github.com/Budibase/budibase/discussions/1913

As for font size and font family - I'd expect to set these on a global level (in future).
